### PR TITLE
FDG-4340: covering edge cases for LQE

### PR DESCRIPTION
**Pull request by dileepk7**

### CODE REVIEW

File: common/src/java/org/apache/hadoop/hive/conf/HiveConf.java [hunk 1]
```diff
@@ -4432,13 +4432,7 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "Fully Qualified class name for concerete impl to be used to enforce queues"),
     JOBNAME_ENRICHER_CLASS("jobname.enricher.class",
         "org.apache.hadoop.hive.ql.propertymodifier.NoEnrichment",
-        "Fully qualified name of class to be used to enrich job name"),
-
-    /**
-     * Properties to support light query engine
-     */
-    LIGHT_QUERY_ENGINE_ENABLED("light.query.engine.enabled", false,
```
Comment (jithendhir92) [inline common/src/java/org/apache/hadoop/hive/conf/HiveConf.java:4440]: Make sure these configs are removed from config buckets after deployment.

**Review by jithendhir92 — COMMENTED** [review]
LGTM

File: beeline/src/java/org/apache/hive/beeline/Commands.java [hunk 2]
```diff
@@ -1422,6 +1422,12 @@ public boolean closeall(String line) {
    * Close the current connection.
    */
   public boolean close(String line) {
+    if (null != beeLine.getSessionHook()) {
+      beeLine.getSessionHook().executePostHook(
+              beeLine.getOpts().getHiveConfVariables(),
```
Comment (mittalnanu) [inline beeline/src/java/org/apache/hive/beeline/Commands.java:1427]: describe what post hook does

**Review by mittalnanu — CHANGES_REQUESTED** [review]
please write test cases

**Review by mittalnanu — APPROVED** [review]
LGTM

**Pull request merged**